tempesta_fw/http_msg.c

```diff
@@ -322,8 +266,8 @@ tfw_http_msg_hdr_close(TfwHttpMsg *hm, int id)
 	*h = hm->parser.hdr;
 
 	TFW_STR_INIT(&hm->parser.hdr);
-	TFW_DBG3("store header w/ ptr=%p len=%lu flags=%x id=%d\n",
-		 h->ptr, h->len, h->flags, id);
+	TFW_DBG3("store header w/ ptr=%p len=%lu eolen=%u flags=%x id=%d\n",
+		 h->ptr, h->len, h->eolen, h->flags, id);
 
 	/* Move the offset forward if current header is fully read. */
 	if (id == ht->off)
```

Comment:
Clang static analyser:
273:9: warning: comparison of integers of different signs: int and unsigned int (-Wsign-compare)

That is not related to patch itself and, warning was here before (and in many places more) and seems not being harmful. May be we should create separated issue with static analyser issues?